packages/obs-backgroundremoval-git/obs-backgroundremoval-git.pacscript

```diff
@@ -37,6 +37,7 @@ build() {
 }
 
 install() {
+  cd build || exit
```

Comment:
```suggestion
  cd build
```

Reply:
![image](https://user-images.githubusercontent.com/10415894/197379381-1e556008-c1ab-407b-bded-4fb2defebb63.png)

Reply:
I am aware. Functions run in an environment where any non-zero exit code fails out completely. So saying `cd build || exit` is redundant.

Reply:
As you wish. I now learned the `git commit --no-verify` option to by-pass the pre-commit hooks.